src/parser.cpp

```diff
@@ -132,8 +133,10 @@ struct QueryGrammar : qi::grammar<Iterator, Command(), Skipper>
                   -('(' >> column_name % ',' >> ')') >> Q("VALUES") >> '(' >>
                   literal % ',' >> ')';
     delete_from = Q("DELETE", "FROM") >> table_name >> where;
+
+    /* Parsing the update command */
     update = Q("UPDATE") >> table_name >> Q("SET") >>
-             ((column_name >> '=' >> literal) % ',') >> -where;
+             column_name >> '=' >> literal >> -where;
```

Comment:
We can have multiple column names in update statement, right? So, please change this back to comma separated list of values.

So.. we will have a vector `colum_name` and `literal`s... Wait, what? Then, what type of `std::vector<?>` is it going to be?

Two ways: Either `std::vector<std::tuple<std::string, LiteralValue>>`, or we need to define a new FUSIONed struct to keep `column_name` and `literal` (perhaps named something like `ast::ColumnValueAssignment`), then we can use `std::vector<ColumnValueAssignment>`. I'd prefer latter option since the former can produce some code a bit ugly to read later (like `get<0>(column_assignment[i])` for getting `column_name` and `get<1>(column_assignment[i])` for getting literal value). It's up to you.

Reply:
Never mind.

> This is what professor has mentioned in the requirement PDF.

Yeah. Just checked PDF. You are right. I probably misinterpreted *one or more records*. Just changing type of `column_name` to `std::string` should suffice.